buildpacks/jvm/src/errors.rs

```diff
@@ -1,48 +1,22 @@
-use crate::{NormalizeVersionStringError, OpenJdkBuildpackError, ValidateSha256Error};
+use crate::openjdk_artifact::HerokuOpenJdkVersionRequirement;
+use crate::{OpenJdkArtifactRequirementParseError, OpenJdkBuildpackError, ValidateSha256Error};
 use buildpacks_jvm_shared::log::log_please_try_again_error;
 use buildpacks_jvm_shared::system_properties::ReadSystemPropertiesError;
 use indoc::formatdoc;
-use libherokubuildpack::download::DownloadError;
 use libherokubuildpack::log::log_error;
 
 #[allow(clippy::too_many_lines)]
 pub(crate) fn on_error_jvm_buildpack(error: OpenJdkBuildpackError) {
     match error {
-        // This mimics the classic behaviour of using download errors as indication for unsupported
```

Comment:
This special download error case can now be removed as we have an inventory file that specifies available versions. Now, if the download fails, it's always a regular download error.